R/package.R

```diff
@@ -178,13 +178,100 @@ pkgdown_config_path <- function(path) {
   )
 }
 
-read_meta <- function(path) {
-  path <- pkgdown_config_path(path)
+read_meta <- function(path, check_path = TRUE) {
```

Comment:
For testing yml files directly, (not use packages), I added this `check_path` argument, that controls whether to look in pkgdown_config_path(). I don't really like this, but it was the quickest way to publish the PR. 

I will happily readjust this.